root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
 </div>
 
 <div class="documentation" data-bind="with: rootField.release">
+  <div class="bubble" data-bind="bubble: $root.titleBubble">
+    <p>
+      [% l('Please see the {doc|style guidelines} for more information.',
+           { doc => { href => doc_link('Style/Titles'), target => '_blank' } }) %]
+    </p>
+  </div>
+
+  <div class="bubble" data-bind="bubble: $root.artistBubble">
+    <p>
+      [% l('Please see the {doc|documentation} and {style|style guidelines} for more information.',
+           { doc => { href => doc_link('Style/Release#Artist'), target => '_blank' },
+             style => { href => doc_link('Style/Artist_Credits'), target => '_blank' } }) %]
+    </p>
+  </div>
+
   <div class="bubble" data-bind="bubble: $root.releaseGroupBubble">
     <!-- ko with: target() && target().releaseGroup -->
-      <p data-bind="html: $data.selectionMessage()"></p>
+      <!-- ko if: $data.gid -->
+        <p data-bind="html: $data.selectionMessage()"></p>
+      <!-- /ko -->
+      <!-- ko ifnot: $data.gid -->
+        <p>
+          [% l('The {doc|release group} groups releases (like different editions of a single album) into a single logical entity.',
+               { doc => { href => doc_link('Release_Group'), target => '_blank' } }) %]
+        </p>
+        <p>
+          [% l('Please see the {doc|style guidelines} for more information.',
```

Comment:
I reused this string in a bunch of places in the hope that it'd make things easier for translators.